sentry_sdk/integrations/celery.py

```diff
@@ -399,6 +399,12 @@ def _get_monitor_config(celery_schedule, app):
             celery_schedule.seconds
         )
 
+        if schedule_unit == "second":
+            logger.warning(
+                "Intervals shorter than one minute are not supported by Sentry Crons."
+            )
+            return {}
```

Comment:
And when a second interval is returned, we print a warning and do not return a monitor config (this is the behavior the backend guys requested, because then they ignore the checkins)